linmod/data.py

```diff
@@ -276,7 +414,57 @@ def main(cfg: Optional[dict]):
 
         print_message(" done.")
     else:
-        print_message("Using cached data.")
+        print_message("Using cached Nextstrain data.")
+
+    if config["data"]["use_usher"]:
+        usher_date = (
+            datetime(
+                config["data"]["forecast_date"]["year"],
+                config["data"]["forecast_date"]["month"],
+                config["data"]["forecast_date"]["day"],
+            )
+            + config["data"]["usher_lag"]
```

Comment:
```suggestion
            + timedelta(days=config["data"]["usher_lag"])
```
With above config edit